assignments/E2.md

```diff
@@ -26,7 +26,7 @@ how to write good tests, and how to develop code using tests as a guide
 
 * Patch 2 adds `username/E2/test.c`, your original testing program, and `username/E2/Makefile`, which builds your testing program as `test` via the default and `test` targets
 
-* Patch 3 adds `username/E2/module.c` and modifies the makefile to build `username/E2/kdlp.ko` via the default and `module` targets
```

Comment:
I think we should only change the file path but not the makefile target. The target is referred to as `module` on line 103 so if we wanted to change it we would need to change it there as well, but I think it is easier to just change it neither place and only make the file path consistent.

Reply:
Updated